src/mds/MDCache.cc

```diff
@@ -12309,6 +12309,13 @@ void MDCache::enqueue_scrub(
 
   mdr->internal_op_finish = cs;
   enqueue_scrub_work(mdr);
+
+  // since recursive scrub is asynchronous, dump minimal output
+  // to not upset cli tools.
+  if (recursive && tag.empty()) {
```

Comment:
What's the output when the `!tag.empty()`? Should they be similar?

Something to think about: assigning a UUID to the tag if it's undefined.

Reply:
`tag path` admin socket command passes a non-empty tag -- we can avoid having that check here is that's what is needed.

Reply:
> What's the output when the !tag.empty()? Should they be similar?

PR updated for dumping minimal output for `scrub_path` and `tag path` asok commands.

> Something to think about: assigning a UUID to the tag if it's undefined.

Yes, maybe. Right now the tags seem to be only used during `ceph-data-scan` tool.